agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -123,6 +123,58 @@ function collect_ara_data {
 	fi
 }
 
+function collect_stockpile_data {
+	# parse config
+	stockpile_path=$(getconf.py stockpile_path stockpile)
+	stockpile_user=$(getconf.py stockpile_user stockpile)
+	stockpile_local_remote_user=$(getconf.py local_remote_user stockpile)
+	stockpile_host_remote_user=$(getconf.py host_remote_user stockpile)
+	stockpile_output_path=$(getconf.py stockpile_output_path stockpile)
+	stockpile_dependencies=$(getconf.py -l stockpile_dependencies packages)
+
+	# install stockpile and its dependencies
+	for package in $stockpile_dependencies; do
+       		which $package &>/dev/null
+        	if [[ $? != 0 ]]; then
+                	debug_log "[$script_name] $package is not installed on the system, installing"
+                	echo "$package is not installed on the system, installing"
+                	yum install $package -y &>/dev/null
+        	fi
+	done
+	if [[ -z "$stockpile_path" ]]; then
+        	stockpile_path=/var/lib/pbench-agent
+	fi
+	if [[ -d "$stockpile_path/stockpile" ]]; then
+        	rm -rf $stockpile_path/stockpile
+	fi
+	git clone https://github.com/redhat-performance/stockpile.git $stockpile_path/stockpile &>/dev/null
+	if [[ $? != 0 ]]; then
+        	echo "Failed to clone stockpile git repository"
+        	error_log "[$script_name] Failed to clone stockpile repository"
+        	exit 1
+	fi
+
+	# set stockpile log file and output paths
+	if [[ -z "$stockpile_log" ]]; then
```

Comment:
Where would this reference to `stockpile_log` be satisfied if not from the config file?

Reply:
@chaitanyaenr, does this question make sense to you?

Reply:
Yes, modified it to place the log file in sysinfo dir along with the other sysinfo log files.